### DISCUSSION TITLE
Where do I find the debug information?

### DISCUSSION THREAD

**Discussion by mzurhorst** [music-assistant/support#2380] [Q&A] [answered]
Hi all,

I have activated the "debug" log level on a few spots in the settings, since MA does show the Player from Home Assistant, but does not play anything.
As far as I can see, the documentation does not tell me where the log file is burried. Also, when browsing through the install folder in HA, it do not find a log file.

What is the appropriate way to get information to identify the root cause?

Thanks & BR, 
  Marcus

**Comment by OzGav** [comment, marked as answer]
MA SETTINGS>>CORE (the engine icon)

**Reply by mzurhorst** [reply]
Ok, thank you.
